concordium-consensus/src/Concordium/GlobalState/Persistent/Trie.hs

```diff
@@ -64,6 +64,10 @@ deriving via Word64 instance FixedTrieKey DelegatorId
 instance FixedTrieKey Bls.PublicKey -- FIXME: This is a bad instance. Serialization of these is expensive.
 instance FixedTrieKey IDTypes.CredentialRegistrationID -- FIXME: this is not the best instance, serialization is expensive.
 
+instance FixedTrieKey BS.ByteString where
+  unpackKey = BS.unpack
+  packKey = BS.pack
```

Comment:
This instance unfortunately does not make sense. But it does make sense for a FixedByteString.

One of the properties instances of FixedTrieKey should satisfy is that the serialized length is always the same.